scio-bigquery/src/main/scala/com/spotify/scio/bigquery/client/TableOps.scala

```diff
@@ -155,26 +155,28 @@ private[client] final class TableOps(client: Client) {
     tableSpec: String,
     selectedFields: List[String] = Nil,
     rowRestriction: String = null
-  ): Schema = {
-    val tableRef = bq.BigQueryHelpers.parseTableSpec(tableSpec)
-    val tableRefProto = TableReferenceProto.TableReference.newBuilder()
-    if (tableRef.getProjectId != null) {
-      tableRefProto.setProjectId(tableRef.getProjectId)
+  ): Schema =
+    Cache.getOrElse(s"""$tableSpec;${selectedFields
+      .mkString(";")};$rowRestriction""", Cache.SchemaCache) {
```

Comment:
not a big deal but use a different separator like `:` in `mkString`? Also can `rowRestriction` have `;`?